legate/core/runtime.py

```diff
@@ -898,11 +1570,12 @@ def _schedule(self, ops):
     def flush_scheduling_window(self):
         if len(self._outstanding_ops) == 0:
             return
+        #self._opLens.append(len(self._outstanding_ops))
         ops = self._outstanding_ops
         self._outstanding_ops = []
         self._schedule(ops)
 
-    def submit(self, op):
+    def submitNew(self, op):
```

Comment:
this was wonchan's new submit function.my scheduling will be 'moved' here once it's cleaned up